holidays/utils.py

```
@@ -435,3 +437,44 @@ def list_supported_financial(include_aliases: bool = True) -> dict[str, list[str
         subdivision codes.
     """
     return _list_supported_entities(EntityLoader.get_financial_codes(include_aliases))
+
+
+def list_long_weekends(
+    instance: HolidayBase,
+    include_non_long_weekends: Optional[bool] = False,
```

Ruff (0.14.0)

Boolean-typed positional argument in function definition

(FBT001)